moz/l10n/fluent/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .parse import fluent_parse, fluent_parse_message
+from .serialize import fluent_astify, fluent_astify_message, fluent_serialize
+
+__all__ = [
+    "fluent_astify",
+    "fluent_astify_message",
+    "fluent_parse",
+    "fluent_parse_message",
+    "fluent_serialize",
```

Comment:
Not really valid, because the result of `fluent_astify_message` is a Fluent pattern, and there is no universally valid canonical serialization of a Fluent pattern, given that the indent level of multiline content varies between message values and attributes, not to mention the selector values where Fluent patterns also show up.